packages/web/public/static/locales/en/error.json

```diff
@@ -3,6 +3,9 @@
   "clientError": "An error occurred on client",
   "invalidPattern": "Invalid pattern",
   "notAuthorized": "You are unauthorized to access this resource",
+  "notAuthorized_female": "",
+  "notAuthorized_male": "",
```

Comment:
Couldn't find a way to disable/change default contexts:

https://i18next-extract.netlify.app/#/faq?id=how-to-deal-with-variable-keys

Reply:
I guess it's better the way you've done it anyway. It's preferable to use non binary words if we can, it's more inclusive.